### PR TITLE
A better toplevel that can do everything in one place

### PR DESCRIPTION
**Pull request by KidusAM**
It includes all the scanner/parser/semant flags and also a complete pipeline for compiling in one go. The input output is like that of the cnet.native executable.

### CODE REVIEW

**Review by Bruk3 — COMMENTED** [review]
LGTM! Let's use single tabs instead of double tabs tho.

File: ccnet [hunk 1]
```diff
@@ -1,42 +1,104 @@
-#!/bin/bash
+#!/bin/python3
```
Comment (Bruk3) [inline ccnet:1]: This doesn't work in mac. Can you try and see if `#!/usr/bin/env python3` works in linux? If so, let's move it to that

**Review by Bruk3 — COMMENTED** [review]
Update: Added a comment on the shebang line for the python script.

File: ccnet (hunk 1 above)
Comment (KidusAM) [reply ccnet:1]: Ya it does

Comment (KidusAM) [reply ccnet:1]: Changed it now

**Pull request merged**